packages/nimble-components/src/card/tests/card.stories.ts

```diff
@@ -0,0 +1,40 @@
+import { html } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import { loremIpsum } from '../../utilities/tests/lorem-ipsum';
+import {
+    createUserSelectedThemeStory,
+    incubatingWarning
+} from '../../utilities/tests/storybook';
+import { buttonTag } from '../../button';
+import { cardTag } from '..';
+
+const overviewText = `The \`nimble-card\` is a container that is designed to contain arbitrary content that is specified by a client
+application. The \`nimble-card\` is intended for grouping related content.`;
+
+const metadata: Meta = {
+    title: 'Incubating/Card',
+    tags: ['autodocs'],
+    parameters: {
+        docs: {
+            description: {
+                component: overviewText
+            }
+        },
+        actions: {}
+    },
+    render: createUserSelectedThemeStory(html`
+        ${incubatingWarning({
+        componentName: 'card',
+        statusLink: 'https://github.com/ni/nimble/issues/296'
+    })}
+        <${cardTag}>
+            <h2>Title</h2>
+            <p>${loremIpsum}</p>
+            <${buttonTag}>Button</${buttonTag}>
```

Comment:
This might seem a bit nitpicky, but we like to have our examples be something that we'd recommend clients doing. So, some things that we'd want to change about this example are:
- Don't use an `<h2>` tag for the title. Instead, you'd want to use a `<span>` or `<div>` styled with nimble tokens. I think it would also be acceptable to remove the title from the example for now since any storybook-specific styling you do now will be negated by the `title` slot that will be coming soon.
- Don't wrap the text in a `<p>`. Instead, use a `<span>` or `<div>` since `<p>` tags have default top/bottom margins that are not tied to nimble tokens.
- Space the elements within the card out using nimble tokens.

I think this could look something like:
```html
<style>
    .title {
        margin-bottom: var(${standardPadding.cssCustomProperty});
        font: var(${titlePlus1Font.cssCustomProperty});
        color: var(${titlePlus1FontColor.cssCustomProperty});
    }

    .body {
        margin-bottom: var(${standardPadding.cssCustomProperty});
    }
</style>
<${cardTag}>
    <div class="title">Title</div>
    <div class="body">${loremIpsum}</div>
    <${buttonTag}>Button</${buttonTag}>
</${cardTag}>
```

Reply:
Not nitpicky at all, thanks for the information! I went ahead and removed the title from the example for now and made the other suggested changes.